src/Storefront/Resources/views/storefront/layout/navigation/offcanvas/item-link.html.twig

```diff
@@ -1,5 +1,7 @@
 {% set isActive = item.category.id == activeId %}
-{% set hasChildren = item.category.visibleChildCount > 0 %}
+{% if not feature('v6.8.0.0') %}
```

Comment:
Should there be a deprecation message here for 6.8 upgrade? And possible something in the upgrade notes how it will be in 6.8.

`hasChildren will be removed` or `has been moved to layout/navigation/offcanvas/categories.html.twig`?

Reply:
I will add this information, it has been moved to the parent template: `layout/navigation/offcanvas/categories.html.twig`

Reply:
@mitelg okay, I added the further information, is it okay like that?